### PR TITLE
2.1.0 Doc Changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Rewrote README.md
- Rewrote setup.md
- Rewrote configuration instructions and usage
- Fixed, typo, structure, and other information flow issues

### CODE REVIEW

File: docs/res/.DS_Store
Comment (cs-yash-patodkar) [inline docs/res/.DS_Store:1]: Please remove this file

File: docs/res/soc_mgmt_edit_json.png
Comment (cs-dev-chaudhary) [inline docs/res/soc_mgmt_edit_json.png:1]: Please replace the screenshot attached with comment. As we have changed the field name from second 'data source' to 'Json Data Source Modules'.

File: docs/setup.md [hunk 1]
```diff
@@ -1,26 +1,47 @@
...
-| Select Field                               | Select the field(Column) of the module which contains the `JSON` data                                                                                                           |
-| Filter Record Which Contains The JSON Data | Add conditions to retrieve only the record meeting the filter conditions. If multiple records match the conditions given, the first record is considered.                       |
+
+The following sections lay out information necessary to customize this widget.
+
+## Record Containing JSON Data
+
+| Fields                                     | Description                                                                                                                                               |
+|--------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Data Source                                | Select the module from which to fetch the data                                                                                                            |
```
Comment (cs-dev-chaudhary) [inline docs/setup.md]: 'Data Source' is renamed to 'Json Data Source Modules'.

File: docs/res/soc_management_light.png
Comment (cs-yash-patodkar) [inline docs/res/soc_management_light.png:1]: Need to change light and dark image, since the incident count doesnt match

File: docs/usage.md [hunk 2]
```diff
@@ -1,257 +1,358 @@
...
+      "value": "320"
+    },
+    {
+      "id": "falsePositives",
+      "title": "False positive",
+      "value": "1K"
+    },
+    {
+      "id": "resolvedAutomatedly",
+      "title": "by Automation )",
```
Comment (cs-dev-chaudhary) [inline docs/usage.md:291]: Please replace "by automation" to "automatedly".

**Review by cs-dev-chaudhary — APPROVED** [review]

**Pull request merged**